plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/DeployImageWizardModel.java

```diff
@@ -104,7 +104,18 @@ public DeployImageWizardModel() {
 	
 	@Override
 	public void dispose() {
+		super.dispose();
 	    DockerConnectionManager.getInstance().removeConnectionManagerListener(this);
+	    ((EnvironmentVariablesPageModel)envModel).dispose();
+	    connection = null;
+	    project = null;
+	    dockerConnection = null;
+	    projects.clear();
+	    volumes.clear();
+	    portSpecs.clear();
+	    if(imagePorts != null) {
+	    	imagePorts.clear();
+	    }
```

Comment:
Ok, I added that for now, but why this field is treated differently from the other (projects, volumes, servicePorts, imageNames) which are initialized with empty lists. I do not see reasons not to do so in this case. Can we initialize this list as well?